docs/api/alerting/list_rule_types.asciidoc

```diff
@@ -63,7 +63,8 @@ context and state in action parameter templates, and a short human readable
 description. When you create a rule in {kib}, it uses this information to prompt
 you for these variables in action parameter editors.
 
-// `alerts`:: TBD
+`alerts`::
+(object) Details about alerts as data documents for this rule type, including any custom mappings.
```

Comment:
I am wondering if we should have an internal version of this API that's used by Kibana that has all this information but the public version of the API doesn't return this? WDYT @XavierM @mikecote

Reply:
That sounds like a good idea, do we know if the current list types API returns `alerts`? If so, we may be too late?

Reply:
I see it in the get rule types response like this:

> ...
{
    "id": ".es-query",
    "name": "Elasticsearch query",
    "category": "management",
    "producer": "stackAlerts",
    "alerts": {
      "context": "stack",
      "mappings": {
        "fieldMap": {
          "kibana.alert.title": {
            "type": "keyword",